sourcegraph/about
# Podcast sponsorship page

Closes #5236 by adding a new landing page for Partner/Sponsored Podcasts.

### To Test
Navigate to `/partner-podcast` and ensure page matches spec & all links are accurate

### Notes
- The mobile Blog list (Under `Related resources`) doesn't match the spec perfectly as I'm re-using a current component for this. Is this something we should leave as it stands in the site or change? I think the image taking up the whole screen width on mobile makes more sense anyway. Let me know!
- Related, blog list image dimensions don't match each other perfectly. Is this worth controlling more or no?
